### DISCUSSION TITLE
Trying to create a non-profit account

### DISCUSSION THREAD

**Discussion by arikatzTK** [community/community#60172] [GitHub Education]
### Hi everyone,

Hi! I've been trying to create a free GitHub account for my non-profit organization for almost three months now to no avail. I submitted a help ticket back in May with the information GitHub asked for to create our non-profit account and still haven't heard back yet. I submitted another ticket to try and see what was going on and no response to that either. Am I doing something wrong? Thank you so much!

**Comment by ilanvaks** [comment]
GitHub usually responds to queries in a reasonable timeframe, definitely check your email spam folders, ensure your organization meets GitHub's eligibility criteria, and double check on all the info you've provided with github if it's correct

**Reply by arikatzTK** [reply]
Interesting! I haven't gotten any emails and I keep checking the support portal but I'm not seeing any responses.

**Comment by sblom** [comment]
I'm facing a very similar situation. I get the initial auto-reponse "thanks. we'll get to this shortly. etc." but here we are months later, and I have seen no further follow-ups for either my initial application or for my follow-up support requests. I feel like I must be doing something wrong.

**Comment by arikatzTK** [comment]
It took a couple months and many support ticket follow-ups (with no
response from GitHub) but we did eventually get our account. I think
they’re just incredibly slow and unresponsive for some reason. I hope
you’re able to get faster results!!
